java-security/pom.xml

```diff
@@ -133,6 +133,12 @@
 			<version>${spring.security.version}</version>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
```

Comment:
does this replace this one?
````
<dependency>
			<groupId>com.github.stefanbirkner</groupId>
			<artifactId>system-rules</artifactId>
			<version>1.19.0</version>
			<scope>test</scope>
		</dependency>
````

Reply:
it's for environment variable testing support in jupiter, as some tests still use junit4 can't replace it completely plus I run into issue when jupiter test didn't work with environment variable and i needed to switch back to junit4 to get it working with surefire plugin